lib/index.js

```diff
@@ -680,6 +691,13 @@ Dataviz.prototype.render = function(results = undefined){
       }
     }
   });
+  // range charts cheat for Min stacked values visibility
+  if(datavizInstance.config.range){
+    if(datavizInstance.config.type.includes('bar')){
+      return document.querySelector('.c3-target-Min').style.display = 'none';
```

Comment:
too general selector, it could break other dataviz instances with different configs - let's use datavizInstance.el to narrow it down

Reply:
maybe it could be better to add a CSS class to hide that
```
.c3-areas-Min {
  display: none;
}
```
but nested inside parent called something like: keen-dataviz-range